Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.yml

```
@@ -296,6 +296,8 @@ script:
         is provided).
     - name: username
       description: Queries users by the samAccountName attribute.
+    - name: samAccountName
```

```suggestion
    - name: sAMAccountName
```
I think args are case sensitive, please check